stylesheets/main.scss

```diff
@@ -1,35 +1,25 @@
 @charset "UTF-8";
 
-// 1. Configuration and helpers
-@import
-  'abstracts/variables',
-  'abstracts/functions',
-  'abstracts/mixins';
+// No need to include partials from the 'abstracts' folder in here top of the file anymore with the @use and @forward module system.
```

Comment:
I think we can drop that comment. It doesn’t have to be part of the boilerplate. :)

Reply:
done.